docker/ubuntu-gtsam-python/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Get the base Ubuntu/GTSAM image from Docker Hub
+FROM dellaert/ubuntu-gtsam:bionic
+
+# Install pip
+RUN apt-get install -y python-pip python-dev
```

Comment:
This should say `python3-pip` and `python3-dev`. Ideally, we should be using `pyenv` to make python version management easy.